### ISSUE TITLE
example do not work

### ISSUE THREAD

**Issue by lksv** [robertleeplummerjr/Leaflet.glify#6]
Example referenced by this repo https://robertleeplummerjr.github.io/Leaflet.glify do not work to me.
I see in console:
```
TypeError: gl is null
vertexShader = gl.createShader(gl.VERTEX_SHADER);
glify.js (line 678, col 9)


TypeError: _instance.polygonLookup is null
feature = _instance.polygonLookup.search(e.latlng.lng, e.latlng.lat);
glify.js (line 799, col 7)
```
Stacktrace:
```
Shapes.prototype.setupVertexShader()glify.js (line 678)
Shapes.prototype.setup()glify.js (line 546)
Shapes()glify.js (line 508)
    settings = Object { type="object"}
L.glify.shapes()glify.js (line 49)
    settings = Object { type="object"}
/Leaflet.glify/()/Leaflet.glify/ (line 45)
    points = Object { type="longString"}
    districts = Object { type="longString"}
wget/</request.onload()/Leaflet.glify/ (line 86)
```
(From FF  44.0)

Chrome (Version 48.0.2564.103 (64-bit)) logs the first error too (with :
```
glify.js:678 Uncaught TypeError: Cannot read property 'createShader' of null
    L.glify.Shapes.Shapes.setupVertexShader @ glify.js:678
    L.glify.Shapes.Shapes.setup @ glify.js:547
    Shapes @ glify.js:509
    L.glify.shapes @ glify.js:49
    (anonymous function) @ (index):45
    request.onload @ (index):86
```

**Comment by lksv** [comment]
Ha! My mistake I do not have WebGL (checked by: chrome://gpu/ )

**Comment by robertleeplummerjr** [comment]
A better means of notification may be ideal.

On Fri, Feb 26, 2016 at 3:16 AM, Lukas Svoboda <notifications@github.com>
wrote:

> Closed #6 <https://github.com/robertleeplummerjr/Leaflet.glify/issues/6>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robertleeplummerjr/Leaflet.glify/issues/6#event-567085313>
> .
>



-- 
Robert Plummer
